ops/terraform/modules/network/main.tf

```diff
@@ -87,4 +89,25 @@ resource "azurerm_private_dns_zone_virtual_network_link" "dns_link" {
   resource_group_name   = var.resource_group
   private_dns_zone_name = azurerm_private_dns_zone.postgresql_dns_zone.name
   virtual_network_id    = azurerm_virtual_network.vnet.id
+  depends_on            = [azurerm_subnet.db-subnet]
+}
+
+# Create private endpoint for SQL server
+resource "azurerm_private_endpoint" "psql_db_pivate_endpoint" {
+  name                = "psql-private-endpoint-${var.env}"
+  location            = var.location
+  resource_group_name = var.resource_group
+  subnet_id           = azurerm_subnet.db-subnet.id
+
+  private_service_connection {
```

Comment:
This is giving me an error on my local. not sure why
<img width="1434" alt="Screenshot 2024-12-11 at 3 54 54 PM" src="https://github.com/user-attachments/assets/30d8b51c-6f20-4cac-92b9-e748a404f777" />